example/lib/src/features/home/views/ui_states/toast_detail_ui_state.dart

```diff
@@ -32,8 +32,8 @@ class ToastDetail with _$ToastDetail {
     @Default(ToastificationStyle.flat) ToastificationStyle style,
     // placement
     @Default(Alignment.topLeft) AlignmentGeometry alignment,
-    @Default('Component updates available.') String title,
-    @Default('Component updates available.') String description,
+    @Default(Text('Component updates available.')) Widget? title,
+    @Default(Text('Component updates available.')) Widget? description,
```

Comment:
Why do we need to make them nullable?

Reply:
To match the intended model struct of the package

Reply:
To clarify Nullable is commonly used by material and Cupertino devs when it comes to building U.I components it provides greater Flexibility to the dev. 

In this case, this allows us to provide more vertical space for the description field, or/and gain a little performance by not having to initialize an empty text Widget.